engine/app/components/citizens_advice_components/sidebar.html.haml

```diff
@@ -0,0 +1,14 @@
+%aside.cads-sidebar
+  .cads-sidebar__content
+    %p.cads-sidebar__title= title
+    - if section_title.url.present?
+      %h3.cads-sidebar__section-title.cads-sidebar__section-title--small
+        = link_to section_title.title, section_title.url
+    - else
+      %h2.cads-sidebar__section-title
+        = section_title.title
+    %nav{ aria: { label: section_title&.title } }
```

Comment:
I wonder if we should make the title mandatory and part of .new's parameters, because we will always want to have an aria-label for the nav.

Then you could get rid of the `&`. As it stands, if there is no title we won't get an `aria-label`.

Reply:
This makes sense. I'm not sure though how to properly write it - in this case should i get rid of the slot as it will only contains the section_title url? A side question (because I just don't know the answer) - if something is required in a slot class (`SectionTitle`), won't it be required on the main object as well (`Sidebar`)?

Reply:
Yeah maybe have title as required and url as optional:

```rb
def initialize(section_title:, section_title_url: nil)
  ...
end
```

(or whatever you want to call the params)

Reply:
On the second point - I don't know for sure but I think if you try and instantiate a slot class without a required param then it'll fail